visual-js/visual-snapshots/package.json

```diff
@@ -0,0 +1,65 @@
+{
+  "name": "@saucelabs/visual-snapshots",
+  "description": "CLI which generates Visual snapshots for a specified document",
+  "version": "0.13.0",
+  "main": "build/index.cjs",
+  "module": "build/index.js",
+  "types": "build/index.d.ts",
+  "license": "MIT",
+  "files": [
+    "build",
+    "README.md"
+  ],
+  "type": "module",
+  "engines": {
+    "node": "^16.13 || >=18"
+  },
+  "typeScriptVersion": "3.8.3",
+  "keywords": [
+    "saucelabs",
+    "visual",
+    "snapshots",
+    "pdf"
+  ],
+  "exports": {
+    ".": {
+      "require": "./build/index.cjs",
+      "import": "./build/index.js",
+      "types": "./build/index.d.ts"
+    },
+    "./package.json": "./package.json"
+  },
+  "scripts": {
+    "build": "tsup",
+    "watch": "tsc-watch --declaration -p .",
+    "lint": "eslint \"{src,test}/**/*.ts\"",
+    "lint-fix": "eslint \"{src,test}/**/*.ts\" --fix"
+  },
+  "dependencies": {
+    "commander": "^12.0.0"
+  },
+  "tsup": {
+    "entry": [
+      "./src/index.ts"
+    ],
+    "dts": true,
+    "outDir": "./build",
+    "format": [
+      "cjs",
+      "esm"
+    ],
+    "noExternal": [
+      "@apollo/client"
+    ]
+  },
```

Comment:
Do we need `tsup` at all? If this will be a CLI tool, I think building by `tsc` will suffice.

Reply:
Switched to `tsc`